1.18/Dockerfile.rhel7

```diff
@@ -60,10 +60,10 @@ RUN yum install -y yum-utils && \
     yum -y clean all --enablerepo='*'
 
 # Copy the S2I scripts from the specific language image to $STI_SCRIPTS_PATH
-COPY $NGINX_VERSION/s2i/bin/ $STI_SCRIPTS_PATH
+COPY 1.20/s2i/bin/ $STI_SCRIPTS_PATH
```

Comment:
This should be "1.18".

Reply:
Copy paste error. Thanks.

Reply:
Fixed by https://github.com/sclorg/nginx-container/pull/229/commits/dc8d1def6ff279fa23d93e7063f063053ebd8daf